mlir/utils/jenkins/Jenkinsfile

```diff
@@ -376,7 +376,7 @@ pipeline {
                            steps {
                                dir('MIOpen') {
                                    getAndBuildMIOpen("--prefix ${WORKSPACE}/MIOpen/MIOpenDeps",
-                                                     '''-DMIOPEN_BACKEND=HIP
+                                                     '''-DMIOPEN_BACKEND=HIP -DMIOPEN_USE_MLIR=Off
```

Comment:
I think cmake booleans are usually spelled in all caps, so `OFF` here is good style - or, in the alternative, `0`